apps/src/code-studio/pd/workshop_enrollment/WorkshopEnrollmentCelebrationDialog.jsx

```diff
@@ -249,6 +297,24 @@ export default function WorkshopEnrollmentCelebrationDialog({
                     {i18n.addToYourCalendar()}
                   </Typography>
                   <div className={style.calendarButtons}>
+                    <LinkButton
+                      text={'Apple'}
+                      ariaLabel={i18n.addToCalendarType({
+                        calendar_type: 'Apple',
+                      })}
+                      type={'secondary'}
+                      color={'black'}
+                      iconLeft={{
+                        iconName: 'brands fa-apple',
+                        iconStyle: 'light',
+                      }}
+                      target="_blank"
+                      href={buildAppleCalendarLink(
+                        workshopSessionInfo,
+                        workshopTitle,
+                        workshopLocation
+                      )}
+                    />
```

Comment:
Since the events will be formatted in an `.ics` file format regardless of the number of sessions, this `LinkButton` can be outside the `hasMultipleSessions` check on line 318.